### PR TITLE
Minimally document snap package

### PR DESCRIPTION
**Pull request by RAOF**

### CODE REVIEW

File: source/installation/with/snap.html.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Installing from the Snap store
+---
+
+Plume is available on the [Snap Store](https://snapcraft.io/plume). The snap package is
+isolated from your host and will automatically update, running any necessary migration
+tasks.
```
Comment (igalic) [inline source/installation/with/snap.html.md:7]: one sentence per line, please
also, an example of how to install from command line would be appreciated

**Review by igalic — CHANGES_REQUESTED** [review]
👀

File: source/installation/with/snap.html.md (hunk 1 above)
Comment (RAOF) [reply source/installation/with/snap.html.md:7]: How's this?

**Review by igalic — APPROVED** [review]
👍

**Review by elegaanz — APPROVED** [review]
Thank you!

**Pull request merged**